components/TARGET_PSA/services/crypto/COMPONENT_SPE/crypto_spe.h

```diff
@@ -65,6 +65,8 @@ extern "C" {
 #define psa_close_key  psa_sec_close_key
 #define psa_hash_clone  psa_sec_hash_clone
 
+#define MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER 1
```

Comment:
doesn't seem to be used

Reply:
it's used for service integration with the crypto library as of https://github.com/ARMmbed/mbed-crypto/pull/59